setup.py

```diff
@@ -33,51 +33,6 @@
 import versioneer
 
 
-class LazyBuildExtCommandClass(dict):
```

Comment:
Can we check for numpy/Cython being installed and fail with a clear error here if they're not?

Reply:
We should remove the `setup_requires` for numpy/Cython as well, since they're the main thing that causes breakage.

Reply:
Hmm, as it currently is, `setup.py develop` is broken. Will need to dig into it if we want to remove even more.

Reply:
Ok, updated.